recipes-core/images/nilrt-runmode-rootfs.bb

```diff
@@ -40,6 +40,12 @@ IMAGE_INSTALL_NODEPS += "\
 	${NI_PROPRIETARY_RUNMODE_PACKAGES} \
 "
 
+COPY_LIC_DIRS_SKIP_PKGS = "\
+        ${NI_PROPRIETARY_COMMON_PACKAGES} \
+	${NI_PROPRIETARY_RUNMODE_PACKAGES} \
+        ni-sysdetails-webservice \
```

Comment:
NIT: check your whitespace.

Reply:
Extra NIT: Put the COPY_LIC_DIRS_SKIP_PKGS values in the same order for runmode/safemode

COPY_LIC_DIRS_SKIP_PKGS = "\
        ${NI_PROPRIETARY_(RUNMODE or SAFEMODE)_PACKAGES} \
        ${NI_PROPRIETARY_COMMON_PACKAGES} \
        ni-sysdetails-webservice \